e2echart/e2e-chart-template.html

```diff
@@ -476,6 +484,9 @@ <h5 class="modal-title">Resource</h5>
         timelineGroups.push({group: "apiserver-shutdown", data: []})
         createTimelineData(apiserverShutdownValue, timelineGroups[timelineGroups.length - 1].data, eventIntervals, isGracefulShutdownActivity, regex)
 
+        timelineGroups.push({group: "api-unreachable", data: []})
+        createTimelineData(isAPIUnreachableFromClientValue, timelineGroups[timelineGroups.length - 1].data, eventIntervals, isAPIUnreachableFromClientActivity, regex)
+
```

Comment:
This is all good, just for some additional info, we're slowly trying to move to using Sippy for the intervals UI (linked in Debug tools on prow job spyglass), and the UI there will now dynamically adapt to new interval "Sources" and let you display them without code changes. It's URL can also be used to link folks directly to the view / filters you're using.